src/main/java/gregtech/api/util/GTUtility.java

```diff
@@ -510,15 +510,7 @@ public static String getColoredTierNameFromTier(byte tier) {
      */
     @Nonnull
     public static String getTierNameWithParentheses(long voltage) {
-        byte tier = getTier(voltage);
-        if (tier < 0) {
-            return "";
-        } else if (tier == 0) {
-            return " (" + GTValues.VN[1] + ")";
-        } else if (tier >= GTValues.VN.length - 1) {
-            return " (MAX+)";
-        }
-        return " (" + GTValues.VN[tier] + ")";
+        return " (" + GTValues.VN[getTier(voltage)] + ")";
```

Comment:
You already have a space between the `EU/t` and the `%s`. This space here is extra.


```suggestion
        return "(" + GTValues.VN[getTier(voltage)] + ")";
```

Reply:
it is now done like this